start-from-zero/Javascript/evaluation.js

```diff
@@ -0,0 +1,118 @@
+const Nav = function(){
+    return `
+    <nav class='navbar'>
+        <a href="index.html"><img src="assets/126-Softvision-logo.png" alt="" class="image"></a>
+        <a class='link-logout' href="#">Logout</a>
+        <div class="container">
+            <div class="row">
+                <div class='link-block-wrapper'>
+                    <a class='link-block' href="#0">Evaluations</a>
+                    <a class='link-block' href="new-evaluation-zero.html">New Evaluations</a>
+                </div>
+            </div>
+        </div>
+    </nav>
+    `;
+}
+
+
+const TableHeadElement = function(options){
+    const tableHead = [];
+    const elements = [{name:'Nume'},{name:'Tehnologie'},{name:'Nivel'}, {name:''}];
+    elements.forEach(function(n){
+        tableHead.push(`<th class='table-head-element'>${n.name}</th>`);
+    })
+    return `
+        ${tableHead.join('')}
```

Comment:
you don't really need the template string here. Just return tableHead.join(....)